STAN.CLIENT/Protocol.cs

```diff
@@ -1,5 +1,7 @@
-// Generated by the protocol buffer compiler.  DO NOT EDIT!
-// source: protocol.proto
+// <auto-generated>
```

Comment:
This includes the PING/PONG support still.

Reply:
re-generated using old proto file